docs/UserGuide.adoc

```diff
@@ -98,31 +119,45 @@ Parameters in *bold* indicate they are mandatory.
     * `s/` - serving size (any number greater than 0)
     * `r/` - rating (scale of 0 to 5)
     * `d/` - difficulty (scale of 0 to 5)
-    * `t/` - tags (seperated by commas)
+    * `t/` - tags (separated by commas)
 
-=== Delete a Recipe — `delete`
-Deletes a recipe from *CookBuddy*.
+Example 1: You can enter `new n/Fried Rice ing/White rice, 1 cup; salt, 1 gram ins/Fry the rice; add salt` into *CookBuddy*, and a new recipe with the following attributes will be added:
+
+    * Name: Fried Rice
+    * Ingredients:
+        ** 1 cup of white rice
+        ** 1 gram of salt
+    * Instructions:
+        . Fry the rice
+        . Add salt
+
+
+=== Deleting a recipe — `delete`
+You can delete a recipe from *CookBuddy*.
```

Comment:
```suggestion
You can delete an existing recipe from *CookBuddy* using the `delete` command.
```

Reply:
or
If you want to delete an existing recipe from *CookBuddy*, you can use the `delete` command.